### DISCUSSION TITLE
getFeaturesDDC Timed out

### DISCUSSION THREAD

**Discussion by srmo2** [xanderfrangos/twinkle-tray#634] [General]
Hello

Fantastic software thank you. 

When using twinkle-tray with 8 monitors connected, no monitors are available for adjustment. With any 7 (or less) of those 8 monitors connected  twinkle-tray functions as designed.

Would it be possible to adjust a setting to allow 8 monitors to function please?

Console output attached
[console.txt](https://github.com/xanderfrangos/twinkle-tray/files/12270628/console.txt)

**Comment by xanderfrangos** [comment]
Yeah, 8 monitors would probably hit the time limit I set for that function. I didn't expect someone to be using that many displays. I guess I need to adjust the timeout to be per-monitor instead of overall.

**Reply by srmo2** [reply]
That would be perfect. Thanks so much for the fast response to the question.

**Reply by xanderfrangos** [reply]
@srmo2 It's fresh out of the oven, but this beta version should help with the issue: https://github.com/xanderfrangos/twinkle-tray/suites/14926504778/artifacts/850828709

The time to read DDC/CI features is now very generous and measured per monitor.

Since this improvement is part of the v1.16.0 beta, this version makes changes to your settings that will not be preserved if you roll back to an earlier version. So if you're concerned about losing your settings, you may want to back up `settings.json` in the `%APPDATA%\twinkle-tray` folder. Also, I'm in the middle of refreshing the settings UI, so some things could be a little busted there (especially on Windows 10).

**Reply by srmo2** [reply]
Thank you very much. Will test again this week and let you know how it goes.

Thanks again.

**Reply by srmo2** [reply]
8 monitors now working ok - thanks for your help
